extension/ui/prefs.ui

```diff
@@ -175,7 +175,7 @@
                                         <property name="valign">end</property>
                                         <property name="wrap">True</property>
                                         <property name="xalign">0</property>
-                                        <property name="label" translatable="yes">Uses 3 finger horizontal swipe to switch workspace on desktop&#xA;and 4 finger to switch windows</property>
```

Comment:
I was using this to add new line here.
Adding newline will make `and` of both lines (line 228 as well) align.
Is that unnecessary ?

Reply:
It definitely created issues if someone resizes the settings window, in which case there are awkward forced breaks. On my screen/font size, "desktop" was on its own line independently. I think it looks fine now and more responsive.